### PR TITLE
Add implementation for contribution view

### PR DESCRIPTION
**Pull request by SwapnilShahGit**
The call to generate the contribution view files need help from @nanuaraj as the command is not waiting for the generation to be completed. This commit also assumes that the user has Python and Gnuplot installed.

I created an issue #39 that deals with adding these two dependencies to the users computer.

### CODE REVIEW

**Comment by SwapnilShahGit** [comment]
TODO: I need to read the config file that has the .git folder and plug it into the contribution view generator appropriately.

**Comment by SwapnilShahGit** [comment]
@hobindar @nanuaraj the config has been pushed onto #35 and I will add the dynamic path once that PR is pushed onto master to avoid repeated code.

**Comment by SwapnilShahGit** [comment]
Please check [#46] before this one.

File: css/mainview.css [hunk 1]
```diff
@@ -116,6 +116,13 @@ body {
     left: 5vh;
 }
 
+#contributionviewicon {
+    background: #306a9c;
+    position: absolute;
+    left:159vh;
```
Comment (hobindar) [inline css/mainview.css]: Missing spacing.

File: mainview.html [hunk 2]
```diff
@@ -5,52 +5,62 @@
...
-        <div class="content">
-            <webview src="./visualiser.html" frameborder="0" nodeintegration></webview>
         </div>
-    </div>
-    <script>window.$ = window.jQuery = require('jquery');</script>
+
+
+    <script>
+        window.$ = window.jQuery = require('jquery');
+
```
Comment (hobindar) [inline mainview.html]: Extra whitespace.

File: mainview.html [hunk 3]
```diff
@@ -5,52 +5,62 @@
...
+                <div class="xbtn">x</div>
             </div>
-            <div class="xbtn">x</div>
-        </div>
-        <div class="content">
-            <div class="custom-wrapper">
-                <webview src="./terminal.html" frameborder="0" nodeintegration></webview>
+            <div class="content">
+                <div class="custom-wrapper">
+                    <webview src="./terminal.html" frameborder="0" nodeintegration></webview>
```
Comment (hobindar) [inline mainview.html]: Let's delete this white space.

**Pull request merged**